experiment/generate_tests.py

```diff
@@ -53,7 +53,7 @@
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
-        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170720-69a2043d
+        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170725-1044fad2
```

Comment:
canary job uses latest, we wait till they pass then push to prod. Kind of staging process here.